### PR TITLE
fix: other user devices UI [AR-1195]

### PR DESCRIPTION
**Pull request by MohammedMokresh**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1195" title="AR-1195" target="_blank">StoryAR-1195</a>  Display Other User’s connected devices
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

the design was not as the mocks in figma

### Solutions

updated the UI , to reflect the figma design

### Attachments (Optional)

Screenshot 2022-08-22 at 12 49 32

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserDevicesScreen.kt [hunk 1]
```diff
@@ -25,19 +28,24 @@ import com.wire.kalium.logic.data.conversation.ClientId
...
 ) {
     val context = LocalContext.current
     val supportUrl = BuildConfig.SUPPORT_URL + stringResource(id = R.string.url_why_verify_conversation)
     LazyColumn(
         state = lazyListState,
-        modifier = Modifier.fillMaxSize()
+        modifier = Modifier
+            .fillMaxSize()
+            .background(
+                color = MaterialTheme.wireColorScheme.surface
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserDevicesScreen.kt:41]: Can't we make this a one-liner?

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-880/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4219889)(4219889)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4219889)(4219889)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt [hunk 2]
```diff
@@ -381,7 +381,7 @@ private fun Content(
 
                             OtherUserProfileTabItem.DEVICES -> {
                                 getOtherUserClients()
-                                OtherUserDevicesScreen(state.otherUserClients)
+                                OtherUserDevicesScreen(state.otherUserClients, state = state)
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt:384]: Why don't we just pass then `state` and refer to `state.otherUserClients` inside `OtherUserDevicesScreen` directly?

**Review by gongracr — APPROVED** [review]
just some small code style comments, but nothing blocking 🥘

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-880/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4220057)(4220057)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4220057)(4220057)-dev-release.apk)

**Pull request merged**